releasetasks/templates/desktop/publish_balrog.yml.tmpl

```diff
@@ -1,5 +1,5 @@
+{% if not release_eta %}
```

Comment:
Yeah, I don't think we should use the same routes or builder names for sure....these builders do different things, and we shouldn't pretend otherwise.